packages/airnode-validator/src/v2/config/config.ts

```diff
@@ -0,0 +1,150 @@
+// TODO: Dynamic checks
+import { z } from 'zod';
+import { oisSchema } from '../ois';
+import { SchemaType } from '../types';
+import { zodDiscriminatedUnion } from '../zod-discriminated-union';
+
+export const triggerSchema = z.object({
+  endpointId: z.string(),
+  endpointName: z.string(),
+  oisTitle: z.string(),
+});
+export type Trigger = SchemaType<typeof triggerSchema>;
```

Comment:
Exporting these types might not be necessary (especially if we end up preserving the types in ois and node packages as I've suggested in the github comment above).

It would make the schemas a bit more compact.